im-sdk-core/src/main/java/com/easemob/im/server/api/metadata/MetadataApi.java

```diff
@@ -33,6 +41,7 @@ public MetadataApi(Context context) {
      * @param username 要被设置用户属性的用户名
      * @param metadata 要设置的属性
      * @return 返回设置的用户属性
+     * @see com.easemob.im.server.api.metadata.user.set.MetadataSetResponse
      */
     public Mono<MetadataSetResponse> set(String username, Map<String, String> metadata) {
```

Comment:
How about return `Mono<Void>`, since we are not simply wrapping the api, we should hide/handle underlying things from developers.